gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/explore.proto

```diff
@@ -32,6 +32,17 @@ message ExploreRequest {
   int32 offset = 21;
 
   string space_id = 22;
+  optional ContextOption context_option = 23;
+}
+
+message ContextOption {
+  oneof option {
+    EntityOption entity_option = 1;
+  }
+}
+
+message EntityOption {
+  bool include_non_live_entities = 1;
```

Comment:
We should ideally get rid of the `optional` flags for start time and end time

Reply:
let me check if those are breaking change or not